libeos-updater-util/flatpak.c

```diff
@@ -490,12 +490,25 @@ action_node_should_be_filtered_out (JsonNode  *node,
   for (iter = filters_object_keys; iter != NULL; iter = iter->next)
     {
       gboolean action_is_filtered_on_this_filter;
+      g_autoptr(GError) local_error = NULL;
 
       if (!action_filter_applies (filters_object,
                                   iter->data,
                                   &action_is_filtered_on_this_filter,
-                                  error))
-        return FALSE;
+                                  &local_error))
+        {
+          if (g_error_matches (local_error,
+                               EOS_UPDATER_ERROR,
+                               EOS_UPDATER_ERROR_UNKNOWN_ENTRY_IN_AUTOINSTALL_SPEC))
+            {
+              g_warning ("%s. Skipping this filter and applying the action anyway, system may be in an inconsistent state from this point forward", local_error->message);
```

Comment:
‘applying the action anyway’ is the opposite of what the design says: actions with unknown filters should not be applied anyway.

Reply:
Ah yes, I must have read the document incorrectly. Thankfully changing this is not too hard (just set `*is_filtered` to `TRUE`)